MobileApplication/MobileApplication/Views/Category/CategoryPage.xaml

```diff
@@ -14,53 +14,59 @@
-                                </FormattedString>
-                            </Label.FormattedText>
-                        </Label>
-                        <Label LineBreakMode="NoWrap" FontSize="16">
-                            <Label.FormattedText>
-                                <FormattedString>
-                                    <Span Text="Budgeted Amount:  "/>
-                                    <Span Text="{Binding CurrentlySpent}" FontAttributes="Bold"/>
-                                </FormattedString>
-                            </Label.FormattedText>
-                        </Label>
-                        <Label LineBreakMode="NoWrap" FontSize="16">
-                            <Label.FormattedText>
-                                <FormattedString>
-                                    <Span Text="Available Amount:  "/>
-                                    <Span Text="{Binding AvailableAmount}" FontAttributes="Bold"/>
-                                </FormattedString>
-                            </Label.FormattedText>
-                        </Label>
-                        <StackLayout.GestureRecognizers>
-                            <TapGestureRecognizer 
-                                NumberOfTapsRequired="1"
-                                Command="{Binding Source={RelativeSource AncestorType={x:Type local:CategoryViewModel}}, Path=ItemTapped}"		
-                                CommandParameter="{Binding .}">
-                            </TapGestureRecognizer>
-                        </StackLayout.GestureRecognizers>
-                    </StackLayout>
-                </DataTemplate>
-            </CollectionView.ItemTemplate>
-        </CollectionView>
-    </RefreshView>
+
+    <ContentPage.Content x:DataType="local:CategoryViewModel">
+        <RefreshView x:DataType="local:CategoryViewModel" Command="{Binding LoadItemsCommand}" IsRefreshing="{Binding IsBusy, Mode=TwoWay}">
+            <StackLayout>
+                <Grid>
+                    <Grid.RowDefinitions>
+                        <RowDefinition Height="Auto"/>
+                    </Grid.RowDefinitions>
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="*" />
+                        <ColumnDefinition Width="*" />
+                        <ColumnDefinition Width="*" />
+                        <ColumnDefinition Width="*" />
+                    </Grid.ColumnDefinitions>
+                    <Label Grid.Column="0" Grid.Row="0" Text="CATEGORY" VerticalTextAlignment="Center"/>
+                    <Label Grid.Column="1" Grid.Row="0" Text="DEDICATED AMOUNT" VerticalTextAlignment="Center"/>
+                    <Label Grid.Column="2" Grid.Row="0" Text="CURRENTLY SPENT" VerticalTextAlignment="Center"/>
+                    <Label Grid.Column="3" Grid.Row="0" Text="AVAILABLE" VerticalTextAlignment="Center"/>
```

Comment:
Future tip. Maybe it's better to use some styling rule (ex. Bold text) if you want the header to be highlighted instead of using hardcoded CAPS.